pontoon/settings/base.py

```diff
@@ -811,6 +811,11 @@ def _default_from_email():
 allowed_hosts = os.environ.get("ALLOWED_HOSTS")
 ALLOWED_HOSTS = allowed_hosts.split(",") if allowed_hosts else []
 
+csrf_trusted_origins = os.environ.get("CSRF_TRUSTED_ORIGINS")
+
+if csrf_trusted_origins:
+    CSRF_TRUSTED_ORIGINS = csrf_trusted_origins.split(",")
```

Comment:
Nit: Can you use the same code pattern here as we already use for the `ALLOWED_HOSTS` above. Both for consistency and to more explicitly communicate the default value.

Reply:
Please also [document](https://github.com/mozilla/pontoon/blob/main/docs/admin/deployment.rst#environment-variables) the new environment variable.

Reply:
done